test/docfx.RegressionTest/RegressionTest.cs

```diff
@@ -174,7 +174,9 @@ private static bool Test(Options opts, string workingFolder)
 
         Compare(testResult, opts, workingFolder, outputPath, baseLinePath);
         Console.BackgroundColor = ConsoleColor.DarkMagenta;
-        Console.WriteLine($"Test Pass {workingFolder}");
+        Console.WriteLine($"Test {(testResult.Succeeded ? "Pass" : "Fail")} {workingFolder}");
+        Console.WriteLine("Test Result Summary:");
+        Console.WriteLine(testResult.ToString());
```

Comment:
nit: `ToString()` here is not necessary

Reply:
Thanks for review, I will update the code.